sympy/external/tests/test_scipy.py

```diff
@@ -24,12 +24,12 @@ def eq(a, b, tol=1e-6):
 
 def test_jn_zeros():
     assert eq(jn_zeros(0, 4, method="scipy"),
-            [3.141592, 6.283185, 9.424777, 12.566370])
+            [3.141592, 6.283185, 9.424777, 12.566370]) == True
```

Comment:
test cases returns `True` so I think it will be better if we write it clearly., what is the output.